lua/package-info/ui/generic/loading-status.lua

```diff
@@ -38,7 +43,7 @@ M.new = function(message)
         notification = nil,
     }
 
-    if nvim_notify then
+    if nvim_notify or snacks_notifier and config.options.notifications then
```

Comment:
Should this maybe be like this 
```suggestion
    if (nvim_notify or snacks_notifier) and config.options.notifications then
```